.github/workflows/rust.yml

```
@@ -62,6 +61,25 @@ jobs:
         if: matrix.os == 'macos-latest'
         run: sudo bash integration-tests/macos.sh
 
+  msrv:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04, macos-latest, windows-latest]
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+
+      - name: Install MSRV toolchain
+        uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: 1.60
```

```suggestion
          toolchain: "1.60"
```

Classic YAML :weary: Nice spot ctz!